File: go/1051.go
// Funcional no beecrowd
package main

import (
	"fmt"
)

func main() {
	var sal float64

	fmt.Scan(&sal)

	if sal >= 0.0 && sal <= 2000.0 {
		fmt.Println("Isento")
	} else if sal <= 3000.0 {
		b := (sal - 2000.0) * .08
		fmt.Printf("R$ %.2f\n", b)
	} else if sal <= 4500.0 {
		b := ((sal - 3000) * .18) + 80
		fmt.Printf("R$ %.2f\n", b)
	} else if sal > 4500.0 {
		b := ((sal - 4500.0) * .28) + 350
		fmt.Printf("R$ %.2f\n", b)
	}
}

File: Python/1 - Iniciante/1051.py
# Funcional no beecrowd
# coding: UTF-8
# Arquivo: 1051

sal = float(input())

if 0 <= sal <= 2000:
    print('Isento')
elif sal <= 3000:
    b = (sal - 2000) * .08
    print(f'R$ {b:.2f}')
elif sal <= 4500:
    b = (sal - 3000) * .18 + 80
    print(f'R$ {b:.2f}')
elif sal > 4500:
    b = (sal - 4500) * .28 + 350
    print(f'R$ {b:.2f}')

# Fim